pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -345,7 +345,7 @@ func (h *NetworkUtilsHandler) GenerateRandomMac() (net.HardwareAddr, error) {
 	return net.HardwareAddr(append(prefix, suffix...)), nil
 }
 
-func (h *NetworkUtilsHandler) CreateTapDevice(tapName string, isMultiqueue bool) error {
+func (h *NetworkUtilsHandler) CreateTapDevice(tapName string, isMultiqueue bool, launcherPID string) error {
```

Comment:
> You seem to pass `launcherPID` but do not use it, making this change hard to understand or make sense.
> I could not see it used anywhere else (e.g. saved in a struct or used to execute something), it is just passed on between functions.

No; it is used to know the MCS categories of the virt-launcher pod.

The commit msg features `have the launcher pid available for future tap device creation` - I can add info there about what the PID will be used for, if you find it helpful.

> 
> The name is also confusing, because I am unclear what it represents. I mean, is this the PID that creates the tap device? From the `CreateTapDevice` perspective, I do not think it needs to know in advance what pid is given, it just need to know that with it, it does something.

I don't know what more can be done to make it explicit that it is the virt-launcher PID.

I need to get those categories, so I can create the tap device using that label (including the categories).

If you prefer (find less confusing) I can just call it PID, but personally think `launcherPID` conveys important information - e.g. that you need info from the launcher.